packages/core/src/meta/ruleset.schema.json

```diff
@@ -172,46 +172,44 @@
         }
       },
       "additionalProperties": {
-        "oneOf": [
-          {
-            "$id": "alias-value",
-            "type": "string",
-            "pattern": "^[$#]",
-            "errorMessage": {
-              "_": "the value of an alias must be a valid JSON Path expression, a reference to the existing Alias optionally paired with a JSON Path expression subset, or contain a valid set of targets",
-              "pattern": "the value of an alias must be a valid JSON Path expression or a reference to the existing Alias optionally paired with a JSON Path expression subset"
-            }
-          },
-          {
-            "type": "object",
-            "properties": {
-              "description": {
-                "type": "string"
-              },
-              "targets": {
-                "type": "array",
-                "minItems": 1,
-                "items": {
-                  "type": "object",
-                  "properties": {
-                    "formats": {
-                      "$ref": "shared#formats"
-                    },
-                    "given": {
-                      "$ref": "alias-value"
-                    }
+        "if": {
```

Comment:
You fancy huh? 💃

Reply:
this is better for validation.
`oneOf` wouldn't generate any meaningful error messages, while this one does.